nbqa/__main__.py

```diff
@@ -452,7 +467,7 @@ def _run_command(
 
 
 def _get_configs(
-    args: argparse.Namespace, kwargs: List[str], tmpdirname: str
+    args: argparse.Namespace, kwargs: List[str], tmpdirname: str, project_root: Path
```

Comment:
Suggestion: kwargs in python is usually given for keyword parameters. So going by the convention shall we rename the kwargs to `cmd_args`?

Reply:
yes, that would be much clearer, thanks!